python/discord/bot.py

```diff
@@ -0,0 +1,149 @@
+import discord
+from discord.ext import commands
+import datetime
+import os
+import sys
+import dotenv
+import asyncio
+import time
+from dotenv import load_dotenv
+dir_path = os.path.dirname(os.path.realpath(__file__))
+parent_dir_path = os.path.abspath(os.path.join(dir_path, os.pardir))
+sys.path.insert(0, parent_dir_path)
+from connection import Topic, connecter
+
+
+load_dotenv()
+
+TOKEN = os.getenv('DISCORD_TOKEN')
+
+mqtt = connecter.create("Discord Bot")
+
+t = Topic()
+
+VELOCITY = 50
+DIRECTION = 50
```

Comment:
Direction should be 0, no? We don't want the car starting off at an angle, if I understand the code correctly.

Reply:
direction is the amount it will turn since they are globals the idea is to not change them